difflint/data/eslint_terse_reporter.js

```diff
@@ -0,0 +1,39 @@
+/*
+ * Format errors for consumption by a lint-diff tool that compares JSHint output
+ * between two revisions of the same file. This reporter is not intended for
+ * human consumption.
+ */
+
+var formatMessage = function (filename, message) {
+    var messageType = "warning";
+
+    // These criteria match the criteria for displaying "error" in the default
+    // reporter for eslint.
+    if (message.fatal || message.severity === 2) {
+        messageType = "error";
+    }
+    
+    return filename + "|" + messageType + "|" + message.message;
+};
+
+var compareMessages = function (filename, messageA, messageB) {
+    var formattedMessageA = formatMessage(filename, messageA);
+    var formattedMessageB = formatMessage(filename, messageB);
+    return formattedMessageA.localeCompare(formattedMessageB);
```

Comment:
Locale compare sorts strings differently in different system locales, according to what the customs for that country and language are, and is meant for human consumption. On the other hand, the default sort function orders strings by Unicode code point no matter what the language is. If the other reporters are using locale compare, then we should probably change them not to.